lib/trace.js

```diff
@@ -23,8 +23,8 @@
  * @constructor
  */
 function Trace(projectId, traceId) {
-  this.projectId = projectId;
-  this.traceId = traceId;
+  this.project_id = projectId;
+  this.trace_id = traceId;
```

Comment:
Do we even need this file anymore? Can't we construct the protobuf class directly?